### PR TITLE
Add questions and results page

### PR DESCRIPTION
**Pull request by yash9263**

### CODE REVIEW

File: styles/pages/question.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.quiz-card {
+  max-width: 440px;
```
Comment (gauravsinhaweb) [inline styles/pages/question.css:2]: you can use rems here.

File: styles/pages/question.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  margin: 0 auto;
+  border-left: 1px solid;
+  border-right: 1px solid;
+
```
Comment (gauravsinhaweb) [inline styles/pages/question.css:6]: remove extra gap here.

File: pages/results.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        </div>
+        <p class="body medium quiz-que">
+          The Godfather movies are based on a novel by what author?
+        </p>
+        <ul class="quiz-options">
+          <li class="quiz-option">ELIA KAZAN</li>
+          <li class="quiz-option">MARIO PUZO</li>
+          <li class="quiz-option">RICHARD CONTE</li>
+          <li class="quiz-option">DARIO FO</li>
+        </ul>
```
Comment (gauravsinhaweb) [inline pages/results.html:80]: use text decoration uppercase instead of hard coding it.

File: pages/results.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        <p class="body medium quiz-que">
+          The Godfather movies are based on a novel by what author?
+        </p>
+        <ul class="quiz-options">
+          <li class="quiz-option wrong">ELIA KAZAN</li>
+          <li class="quiz-option correct">MARIO PUZO</li>
+          <li class="quiz-option">RICHARD CONTE</li>
+          <li class="quiz-option">DARIO FO</li>
+        </ul>
+      </article>
```
Comment (gauravsinhaweb) [inline pages/results.html:66]: good use of semantic tags!

**Review by Maianki — APPROVED** [review]
Everything looks good. You have followed all good practices just have a small suggestion.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes. UI looks clean!

**Pull request merged**